Vaults/Nodsoft.MoltenObsidian.Vaults.Ftp/VaultDependencyInjectionExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Text.Json;
+using FluentFTP;
+using JetBrains.Annotations;
+using Microsoft.Extensions.DependencyInjection;
+using Nodsoft.MoltenObsidian.Manifest;
+using Nodsoft.MoltenObsidian.Vault;
+
+namespace Nodsoft.MoltenObsidian.Vaults.Ftp;
```

Comment:
Change namespace to `Microsoft.Extensions.DependencyInjection`, for a more conventional extension methods approach.